### PR TITLE
[5779] Use markdown and html rather than translations for content pages

### PR DESCRIPTION
**Pull request by defong**
### Context

Content pages

### Changes proposed in this pull request
Use markdown and html rather than translations for content pages

### Guidance to review
Nothing is broken
No missing translations

https://register-pr-3575.test.teacherservices.cloud/accessibility
https://register-pr-3575.test.teacherservices.cloud/privacy-notice
https://register-pr-3575.test.teacherservices.cloud/data-sharing-agreement

### Important business

- [x] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [x] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [x] Do we need to send any updates to DQT as part of the work in this PR?
- [x] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: app/controllers/root_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RootController < ApplicationController
```
Comment (carlosmartinez) [inline app/controllers/root_controller.rb:3]: is `Root` the right noun to describe whatever this resource is?

**Comment by carlosmartinez** [comment]
please link trello ✊🏽

**Comment by andrewscrivener** [comment]
Fine from a content and design perspective @defong 👍

**Comment by defong** [comment]
[Use Markdown and HTML rather than translations for content pages](https://trello.com/c/8kvGJbTk/5779-use-markdown-and-html-rather-than-translations-for-content-pages)

File: app/controllers/root_controller.rb (hunk 1 above)
Comment (defong) [reply app/controllers/root_controller.rb:3]: Suggestions welcome :pray: 

from  route.rb
```ruby
root to: "root#start"
```

from usage
```
root_path
```

actually location is also `/`

I did consider `HomeController` but felt stumped by 
```
views/home/home.html.erb
views/home/start.html.erb
```

Comment (defong) [reply app/controllers/root_controller.rb]: Discussed and we settled on `LandingPage` vs `Home` vs `Dashboard`

**Review by carlosmartinez — APPROVED** [review]

**Pull request merged**